src/game_objects/construct.h

```diff
@@ -10,6 +10,7 @@
 class Construct : public GameObject {
 protected:
     btBoxShape *hit_box = new btBoxShape(btVector3(0.5f, 0.5f, 0.5f));
+    btCylinderShape *goal_hit_box = new btCylinderShape(btVector3(0.005f, 0.5f, 0.005f));
```

Comment:
We should move hit boxes outside of the base class and make them specific to the construct themselves, since more traps will need more specific boxes that we need to fine tune.